5001-6000/LIT5900DersanaMikaelTheo.xml

```diff
@@ -44,7 +44,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <keywords>
                     <term key="ChristianLiterature"/>
                     <term key="Homily"/>
-                    <term key="Religion"/>
+                  <!--  <term key="Religion"/>-->
```

Comment:
In principle category "Miracle" would fit to Dersana Mika'el, also "Dersana Gabre'el".